rorapi/settings.py

```diff
@@ -180,7 +180,12 @@
                            'grid-{}'.format(GRID['VERSION']))
 GRID['GRID_ZIP_PATH'] = os.path.join(GRID['DIR'], 'grid.zip')
 GRID['GRID_JSON_PATH'] = os.path.join(GRID['DIR'], 'grid.json')
-GRID['ROR_ZIP_PATH'] = os.path.join(GRID['DIR'], 'ror.zip')
-GRID['ROR_JSON_PATH'] = os.path.join(GRID['DIR'], 'ror.json')
+
+ROR = {'VERSION': '2019-09-19'}
```

Comment:
I would rename this to ROR_DUMP, to be more specific about what this group of settings is.

Reply:
Done